dev/sparktestsupport/modules.py

```diff
@@ -488,12 +495,10 @@ def __hash__(self):
     python_test_goals=[
         # doctests
         "pyspark.streaming.util",
```

Comment:
BTW, do you plan to do it for doctests later? Doing it in a separate PR is fine. I'm just curious about your plan.

Reply:
Frankly, not yet, but I'd like to start to do some investigation. This unittestPR is started when I forgot many times to add the unittests in module. ^_^

And the `doctests` discover is a bit special, it's not subclass of doctest (just like the unitest), so looks like there are 2 potential solution:

1. Flag based. Add some flag like `_doc_test = True` in every test we want to test.
2. Special name based. Discover which module contains `_doctest` method (we should rename _test to _doctest to make doc test name unique).

Reply:
BTW, do you think the footer method (excute doctest or unitests in __main__ with repeat code in many files) in python code is a good way enough? Do we consider do some re-arch (or to say, already have some work)on testing framework?